qcodes/instrument/parameter.py

```diff
@@ -27,7 +27,7 @@
 - :class:`.ParameterWithSetpoints` is intended for array-values parameters.
     This Parameter class is intended for anything where a call to the instrument
     returns an array of values.
-    `This notebook <../examples/writing_drivers/Simple-Example-of-ParameterWithSetpoints.ipynb>`_.
+    `This notebook <../../examples/writing_drivers/A-ParameterWithSetpoints-Example-with-Dual-Setpoints.ipynb>`_.
```

Comment:
shouldn't it be this https://github.com/QCoDeS/Qcodes/blob/master/docs/examples/Parameters/Simple-Example-of-ParameterWithSetpoints.ipynb ?

Reply:
or just both :)

Reply:
Maybe both. I was also confused about what would be a better choice.

Reply:
the SimpleExample is definitely the preferred choice. But let's have both.